### PR TITLE
docs: add migration guide and maintenance tasks section

### PR DESCRIPTION
**Pull request by salasberryfin**
# Description

A guide on how to migrate to the new import controller has been added under a new section `Tasks` -> `Maintenance`. This includes how to use the [migration script](https://github.com/rancher/turtles/pull/574) or apply changes manually. Other similar guides have been moved under this section as well.

### CODE REVIEW

**Review by alexander-demicev — APPROVED** [review]
thanks!

File: docs/tasks/maintenance/import_controller_upgrade.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+## Context
+
+When Turtles imports a CAPI cluster into Rancher, a number of Kubernetes resources must be created to represent the CAPI cluster in Rancher. The main resource is the one representing the CAPI cluster as a Rancher cluster, so it can be viewed and managed via Rancher:
+```
+clusters.provisioning.cattle.io
+```
+We also call this resource a `v1` cluster. If you were using Turtles before v0.9.0, the default import controller was based on generating this resource for every CAPI cluster that is imported into Rancher. The controller we will be migrating to was disabled behind a feature gate `managementv3_cluster`.
+
+With the new Turtles release, the `v3` import controller is no longer behind a feature gate and will be the default import logic. This means we are moving to a different strategy for importing clusters, and switching to creating a different type of Rancher cluster resource:
```
Comment (furkatgofurov7) [inline docs/tasks/maintenance/import_controller_upgrade.md:15]: If we are already sure, we can highlight it clearly from which release and onwards

**Comment by salasberryfin** [comment]
Thanks @furkatgofurov7 @alexander-demicev. The migration docs now refer explicitly to v0.9.0 as the first version in which the default import controller is based on v3 clusters.

File: docs/tasks/maintenance/early_adopter_upgrade.md
Comment (yiannistri) [inline docs/tasks/maintenance/early_adopter_upgrade.md:1]: Unrelated to this PR's changes but we use number 3 twice in the list of steps. Perhaps we can include a fix for this as part of this PR?

**Review by yiannistri — APPROVED** [review]

**Review by furkatgofurov7 — APPROVED** [review]

**Pull request merged**